module/applications/actor/base-sheet.mjs

```diff
@@ -73,12 +73,28 @@ export default class ActorSheetEd extends ActorSheet {
     // Owned Item management
     html.find( ".card__ability .take-strain" ).click( this._takeStrain.bind( this ) );
 
-    // Owned Item management
-    html.find( ".card__ability .take-strain" ).click( this._takeStrain.bind( this ) );
-
     // Owned Item management
     html.find( ".item-delete" ).click( this._onItemDelete.bind( this ) );
 
+    // Actor Sheet Buttons
```

Comment:
I would suggest to add a data attribute to these elements on the basis of which the appropriate function is chosen. So you only need to find one class, like ".rollable". So, e.g., `<span class=".rollable" data-action="recovery">`. And the function is something like:
```javascript
html.find( ".rollable" ).click( event => {
  const action = event.currentTarget.dataset.action;
  actionMapping = {
    "recovery": this._onRecoveryRoll( event ),
    "takeDamage": this._onTakeDamage( event ),
    // or, if only one line:
    "jumpUp": this.actor.jumpUp(),
    // etc....
  };
} );
```

Reply:
Clever